code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -184,10 +184,9 @@ Credit where due:
 	return FALSE
 
 /datum/game_mode/clockwork_cult/check_finished()
-	var/obj/structure/destructible/clockwork/massive/celestial_gateway/G = GLOB.ark_of_the_clockwork_justiciar
-	if(G && !GLOB.ratvar_awakens) // Doesn't end until the Ark is destroyed or completed
+	if(GLOB.ark_of_the_clockwork_justiciar && !GLOB.ratvar_awakens) // Doesn't end until the Ark is destroyed or completed
```

Comment:
was this misspelled purposefully? its supposed to be justicar except its spelt justiciar here in da code

Reply:
it's spelled the same way here as in the deleted line, so it's intentional in the sense that it must be spelled this way to function